### PR TITLE
Update LibOvrIntegration to Oculus SDK 0.6.0.1 beta

### PR DESCRIPTION
**Pull request by Squareys**
Hi @mosra !

Just to let you know that this is being worked on.

TODOs:
 - [x] Fix compilation with 0.6.0.1
 - [x] Changes related to queue ahead (none, magically works in the background ;) )
 - [x] Wrap the performance HUD

I will continue work on this tomorrow.

### CODE REVIEW

**Comment by mosra** [comment]
Hey, thanks! Oh and enjoy the holidays :)

**Comment by Squareys** [comment]
@mosra I did not try if the performance hud works, it was pretty simple straight forward code, though. Maybe later I will try it out in the example (needs adaption to the removed enums anyway). Other than that, this is ready from my point of view, you may review ;)

File: src/Magnum/LibOvrIntegration/Hmd.h [hunk 1]
```diff
@@ -337,6 +337,15 @@ class MAGNUM_LIBOVRINTEGRATION_EXPORT Hmd {
...
         }
 
+        /**
+         * @brief Enable/Disable the performance HUD
+         *
+         * Performance HUD enables the HMD user to see information critical to
+         * the real-time operation of the VR application such as latency timing
+         * and CPU & GPU performance metrics.
+         */
+        void setPerformanceHudMode(const PerformanceHudMode& mode) const;
```
Comment (mosra) [inline src/Magnum/LibOvrIntegration/Hmd.h]: This is just an four-byte enum, right? I'd pass it directly by value, passing it by reference costs 8 bytes (twice as much). Other than that it's ready for merge :)

Comment (Squareys) [inline src/Magnum/LibOvrIntegration/Hmd.h]: Alright, makes sense :)

**Comment by mosra** [comment]
Thank you!

**Pull request merged**